spec/interactors/knight_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Knight do
+  describe '::call' do
```

Comment:
That it's a class method, opposed to an instance method which would be `#call`.


Based on the notation in the docs: https://ruby-doc.org/core-2.2.0/Array.html

<img width="257" alt="screenshot 2017-03-24 14 43 37" src="https://cloud.githubusercontent.com/assets/79998/24312893/50875476-10a0-11e7-9e77-62ddbb6cff5e.png">